REFERENCE.md

```diff
@@ -25,14 +25,22 @@
 
 ### <a name="hiera"></a>`hiera`
 
-== Class: hiera
-
-This class handles installing the hiera.yaml for Puppet's use.
-
-=== Parameters:
-
-  See README.
-
+Example hiera data for the backend_options hash:
+
+backend_options:
+ json:
+   datadir: '/etc/puppetlabs/puppet/%{environment}/jsondata'
+ redis:
+   password: clearp@ssw0rd        # if your Redis server requires authentication
+   port: 6380                     # unless present, defaults to 6379
+   db: 1                          # unless present, defaults to 0
+   host: db.example.com           # unless present, defaults to localhost
+   path: /tmp/redis.sock          # overrides port if unixsocket exists
+   soft_connection_failure: true  # bypass exception if Redis server is unavailable; default is false
+   separator: /                   # unless present, defaults to :
+   deserialize: :json             # Try to deserialize; both :yaml and :json are supported
```

Comment:
This will not display correctly.  I guess you can check spacing(I see 1 / 3 spaces but would expect 2 / 4 to be consistent) and use triple-backticks before and after the fragment.

Reply:
ok, will do